### ISSUE TITLE
running issue

### ISSUE THREAD

**Issue by cyanebay** [ddlee-cn/MuLUT#7]

**Comment by cyanebay** [comment]
Can you provide the demosaicking pipeline for the paper? thank you.

**Comment by CarrieWang0319** [comment]
Based on your instructions, I downloaded datasets such as set5 and set14 and placed them in the dataset \ SRBenchmark directory. However, I found that the downloaded dataset did not contain LR_bicubic/X2/*. png, only HR/*. png. When I tried to test, I encountered an error::The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "4_test_lut.py", line 340, in <module>
    etr.run()
  File "4_test_lut.py", line 259, in run
    psnr_ssim_s = pool.map(self._worker, list(range(len(self.files))))
  File "E:\python\anaconda\envs\yolov5\lib\multiprocessing\pool.py", line 266, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "E:\python\anaconda\envs\yolov5\lib\multiprocessing\pool.py", line 644, in get
    raise self._value:,
 may I ask why? If possible, I would greatly appreciate it
